src/htm/algorithms/AnomalyLikelihood.hpp

```diff
@@ -1,18 +1,18 @@
-#ifndef NUPIC_ALGORITHMS_ANOMALY_LIKELIHOOD_HPP_
-#define NUPIC_ALGORITHMS_ANOMALY_LIKELIHOOD_HPP_
+#ifndef HTM_ALGORITHMS_ANOMALY_LIKELIHOOD_HPP_
+#define HTM_ALGORITHMS_ANOMALY_LIKELIHOOD_HPP_
 
-#include <nupic/types/Serializable.hpp>
-#include <nupic/types/Types.hpp>
-#include <nupic/utils/MovingAverage.hpp>
-#include <nupic/utils/SlidingWindow.hpp>
-#include <nupic/utils/Log.hpp>
+#include <htm/types/Serializable.hpp>
+#include <htm/types/Types.hpp>
+#include <htm/utils/MovingAverage.hpp>
+#include <htm/utils/SlidingWindow.hpp>
+#include <htm/utils/Log.hpp>
 
 #include <string>
 #include <vector>
 #include <cmath>
 
 /**
-Note: this is an implementation from python in nupic repository.
+Note: this is an implementation from python in htm repository.
```

Comment:
nitpick, this should not have changed, since it refers to nupic.py which we are not renaming.  This comment should probably say the URL: https://github.com/htm-community/nupic.py

Reply:
Will be fixed in PR #526